dashboard/lti_new.py

```diff
@@ -7,6 +7,7 @@
 import django.contrib.auth
 from django.conf import settings
 from django.contrib.auth.models import User
+from dashboard.models import User as MyLAUser
```

Comment:
Looks like there's already a `MylaUser` alias. Can you use that one?

Reply:
Removed!